### PR TITLE
Move remix alert

### PR DESCRIPTION
**Pull request by aninternetof**
This moves the remix message alert box to be in line with the program name. It also tweaks the wording of the message.

### CODE REVIEW

File: src/components/Workspace.js [hunk 1]
```diff
@@ -437,71 +437,20 @@ class Workspace extends Component {
...
-        changeReadOnly(false);
-        this.createWorkspace();
-      });
-  }
 
   render() {
     const { children, code, rover } = this.props;
 
     return (
-      <Box m={1}>
```
Comment (aninternetof) [inline src/components/Workspace.js:468]: Removing this padding allowed me to make the workspace line up with the program title bar without also adding the same padding to the title bar, which it didn't really need.
I don't think it makes the workspace look too tight, but you can let me know what you think.

File: src/containers/MissionControl.js [hunk 2]
```diff
@@ -28,132 +30,233 @@ import Indicator from '@/components/Indicator';
...
+      intl,
+      location,
+      code,
+    } = this.props;
+
+    const {
+      open,
+    } = this.state;
+
+    const readOnlyTitle = intl.formatMessage({
```
Comment (cabarnes) [inline src/containers/MissionControl.js:98]: This needs to be added to the translation files

File: src/containers/MissionControl.js [hunk 3]
```diff
@@ -28,132 +30,233 @@ import Indicator from '@/components/Indicator';
...
+              {
+                code.isReadOnly ? (
+                  <Grid item>
+                    <Box m={0}>
+                      <Alert
+                        severity="info"
+                        action={(
+                          <Button color="primary" variant="contained" size="huge" onClick={this.remix}>
+                            <FormattedMessage
+                              id="app.workspace.remix"
```
Comment (cabarnes) [inline src/containers/MissionControl.js:164]: This needs to change to `app.mission_control.remix`

File: src/containers/MissionControl.js [hunk 4]
```diff
@@ -28,132 +30,233 @@ import Indicator from '@/components/Indicator';
...
+                              defaultMessage="Remix"
+                            />
+                          </Button>
+                        )}
+                      >
+                        <AlertTitle>
+                          {`${readOnlyTitle} ${code.ownerName}.`}
+                        </AlertTitle>
+                        <FormattedMessage
+                          id="app.workspace.read_only_content"
```
Comment (cabarnes) [inline src/containers/MissionControl.js:175]: This needs to change to `app.mission_control.read_only_content`

**Comment by aninternetof** [comment]
That graphite error on remix is from the Sumo logger. If you comment out the logging, it should work.
The error has always been there when running locally; we can figure out some way to supply a Sumo endpoint in the local environment.

**Comment by cabarnes** [comment]
Or can we check the environment and not call the sumo logger locally?

**Pull request merged**